Makefile

```diff
@@ -1,40 +1,36 @@
 #!/usr/bin/make -f
 
-
-BRANCH         := $(shell git rev-parse --abbrev-ref HEAD)
-COMMIT         := $(shell git log -1 --format='%H')
-BUILD_DIR      ?= $(CURDIR)/build
-DIST_DIR       ?= $(CURDIR)/dist
+PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
+PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
+export VERSION := $(shell echo $(shell git describe --always --match "v*") | sed 's/^v//')
+export TMVERSION := $(shell go list -m github.com/tendermint/tendermint | sed 's:.* ::')
+export COMMIT := $(shell git log -1 --format='%H')
 LEDGER_ENABLED ?= true
-TM_VERSION     := $(shell go list -m github.com/tendermint/tendermint | sed 's:.* ::')
-DOCKER         := $(shell which docker)
-PROJECT_NAME   := umee
-HTTPS_GIT      := https://github.com/umee-network/umee.git
+BINDIR ?= $(GOPATH)/bin
+BUILDDIR ?= $(CURDIR)/build
+DIST_DIR ?= $(CURDIR)/dist
 MOCKS_DIR = $(CURDIR)/tests/mocks
+HTTPS_GIT := https://github.com/umee-network/umee.git
+DOCKER := $(shell which docker)
+DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.0.0-rc8
+PROJECT_NAME := umee
 
-###############################################################################
-##                                  Version                                  ##
-###############################################################################
+# RocksDB is a native dependency, so we don't assume the library is installed.
+# Instead, it must be explicitly enabled and we warn when it is not.
+ENABLE_ROCKSDB ?= false
 
-ifeq (,$(VERSION))
-  VERSION := $(shell git describe --exact-match 2>/dev/null)
-  # if VERSION is empty, then populate it with branch's name and raw commit hash
-  ifeq (,$(VERSION))
-    VERSION := $(BRANCH)-$(COMMIT)
-  endif
-endif
 
 ###############################################################################
 ##                                   Build                                   ##
 ###############################################################################
 
-build_tags = netgo
 
-#  experimental feature 
+#  experimental feature
 ifeq ($(EXPERIMENTAL),true)
 	build_tags += experimental
 endif
 
+build_tags = netgo
```

Comment:
`build_tags = netgo` should be above the 
```#  experimental feature
ifeq ($(EXPERIMENTAL),true)
	build_tags += experimental
endif
```

Reply:
yes, thanks. updated